CHANGELOG.md

```diff
@@ -1,5 +1,108 @@
 # Changelog
 
+## v1.11.0 (03.02.2021)
+
+### Enhancements
+
+#### :gear: Control Request Timeouts ([PR #5692](https://github.com/DevExpress/testcafe/pull/5692))
+
+New CLI options allow you to control the timeout for requests. If a request is not met with a response within the specified period, TestCafe throws an error. The CLI options are:
+
+* [--ajax-request-timeout](./docs/articles/documentation/reference/command-line-interface.md#--ajax-request-timeout-ms) controls the delay for fetch/XHR requests timeout
+* [--page-request-timeout](./docs/articles/documentation/reference/command-line-interface.md#--page-request-timeout-ms) sets the timeout for receiving webpages.
+
+Read the documentation for the corresponding configuration properties:
+
+* [ajaxRequestTimeout](./docs/articles/documentation/reference/configuration-file.md#ajaxrequesttimeout)
+* [pageRequestTimeout](./docs/articles/documentation/reference/configuration-file.md#ajaxrequesttimeout).
+
+The options are available in the [runner.run Method](./docs/articles/documentation/reference/testcafe-api/runner/run.md).
+
+#### :gear: Control Browser Initialization Timeout ([PR #5720](https://github.com/DevExpress/testcafe/pull/5720))
+
+This release introduces an option to control browser initialization timeout. This timeout controls the time browsers have to connect to TestCafe before an error is thrown. You can control this timeout in one of the following ways:
+
+* [--browser-init-timeout](./docs/articles/documentation/reference/command-line-interface.md#--browser-init-timeout-ms) CLI option
+
+```sh
+testcafe chrome my-tests --browser-init-timeout 180000
+```
+
+* [browserInitTimeout](./docs/articles/documentation/reference/configuration-file.md#browserinittimeout) configuration option
+
+```json
+{
+    "browserInitTimeout": 180000
+}
+```
+
+* [runner.run Method](./docs/articles/documentation/reference/testcafe-api/runner/run.md) parameter
+
+```js
+runner.run({ "browserInitTimeout": 180000 })
+```
+
+This setting sets an equal timeout for local and [remote browsers](./docs/articles/documentation/guides/concepts/browsers.md#browsers-on-remote-devices).
+
+#### Improved the `Unable To Establish Browser Connection` Error Message ([PR #5720](https://github.com/DevExpress/testcafe/pull/5720))
+
+TestCafe raises this error when at least one local or remote browser wasn't able to connect. The error message now includes the number of browsers that have not established a connection.
+
+TestCafe raises a warning if low system performance is causing the connectivity issue.
+
+#### :gear: An Option to Retry Requests for the Test Page ([PR #5738](https://github.com/DevExpress/testcafe/pull/5738))
+
+If the first request for the tested webpage is not satisfied, TestCafe can now retry this request.
```

Comment:
I think it's simply "responded", not "satisfied".

I would say "a request is satisfied" if I meant "The response has been received and the sender is satisfied with the response"... 😄

Reply:
If a tested webpage wasn't served after the first request, TestCafe can now retry this request.